<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1208</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(38, 38, 38);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>471</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 78pt &quot;Helvetica&quot;;</string>
    </property>
    <property name="text">
     <string>FrugalDoodle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>600</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(83, 83, 83);</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>461</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);
font: 20pt &quot;Baskerville&quot;;</string>
      </property>
      <property name="text">
       <string>Choose a WiFi Network:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>241</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string>Wifi 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wifi 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wifi 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wifi 4</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);
font: 20pt &quot;Baskerville&quot;;</string>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>240</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 20pt &quot;Baskerville&quot;;</string>
      </property>
      <property name="text">
       <string>Scan the QR Code</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>600</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1208</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
